assayist/processor/configuration.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+import os
+import logging
+
+
+class Config(object):
+    """The base Assayist Processor configuration."""
+
+    brew_url = 'https://brewhub.engineering.redhat.com/brewhub'
```

Comment:
I think this should be `koji_url`, or even `koji_profile` so we can `koji.load_config()` it (if we can arrange for /etc/koji.conf.d/ to be populated correctly).

Reply:
Good point. I changed the PR to use profiles.